setup.cfg

```diff
@@ -1,4 +1,13 @@
+[coverage:run]
+source = openslides
+
+[coverage:html]
+directory = development/htmlcov
```

Comment:
I understand the folder 'development' as a folder like ~/.config/openslides where all files are, that are needed to run openslides. So this folder can be moved from one instance to another. Of cause, in this case the htmlcov does no harm, but it is not needed neither. For that reason I would like it to be in a different place. What about ```/tmp```?